//离开公会
message DseGuildQuit
{
	optional int32 res = 1;
}
